_includes/code/components/table.html

```diff
@@ -1,3 +1,3 @@
 {% capture table %}
-  {% fractal_component table %}
-{% endcapture %}{{ table | strip }}
\ No newline at end of file
+  {% library_component usa-table %}
+{% endcapture %}{{ table | strip }}
```

Comment:
I noticed we don't show the default table styles seen [here]9https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/al-ld-remove-compare/?path=/story/components-table--default). Should we include it?